erpnext/patches/v15_0/update_uae_zero_rated_fetch.py

```diff
@@ -0,0 +1,9 @@
+import frappe
+from frappe.custom.doctype.property_setter.property_setter import make_property_setter
+
+
+def execute():
+	if not frappe.db.get_value("Company", {"country": "United Arab Emirates"}):
+		return
+
+	make_property_setter("Sales Invoice Item", "is_zero_rated", "fetch_if_empty", 1, "Check")
```

Comment:
custom field shouldn't be updated using a property setter. Why not directly update the custom field?